internal/handlers/tigris/tigrisdb/query.go

```diff
@@ -85,7 +88,19 @@ func (tdb *TigrisDB) QueryDocuments(ctx context.Context, param *FetchParam) ([]*
 		res = append(res, doc.(*types.Document))
 	}
 
-	return res, iter.Err()
+	err = iter.Err()
+
+	switch {
+	case err == nil:
+		fallthrough
+	case IsInvalidArgument(err):
+		// Skip errors from filtering invalid types
+		break
```

Comment:
@noisersup Why do we do that? IIUC, we just hide what could be a real error

Reply:
InvalidArgument error occures while filtering the schema with wrong type. For example if `v` is a string, but we use filter `{"v":42}` we get this error. MongoDB on this scenario result with no error, as it just means that filter don't match the value in database

Reply:
ic, thanks